src/functions.php

```diff
@@ -149,7 +167,10 @@ function after($it, $that)
 function run($command)
 {
     $server = Context::get()->getServer();
-    $command = Context::get()->getEnvironment()->parse($command);
+    $command = env()->parse($command);
+    $workingPath = workingPath();
+
+    $command = "cd $workingPath && $command";
```

Comment:
Interestingly enough, it does because `workingPath()` calls `get` on the environment, which does the following before returning the value `return $this->parse($value);`.